src/components/login/Login.tsx

```diff
@@ -148,6 +158,16 @@ export default class Login extends Component<LoginProps, LoginFormState> {
                             </a>
                         )
                     })}
+
+            { !localStorage.isDashboardLoggedIn &&  <InfoColourBar
```

Comment:
As `localStorage.isDashboardLoggedIn` will return string, check should be something like
```
localStorage.isDashboardLoggedIn !== 'true'
```